README.md

```diff
@@ -1,151 +1,36 @@
-- Version 
-
-See: [ModuleTemplate Reference](https://github.com/kyma-project/lifecycle-manager/blob/main/docs/contributor/resources/03-moduletemplate.md)
+This repository contains ModuleTemplate CRs for modules contributed by the Kyma community. You can install community modules in Kyma clusters using Kyma dashboard, Kyma CLI, or kubectl. Community modules offer flexibility and innovation beyond the fully-managed modules offered SAP BTP, Kyma runtime. For more information, see [Community Modules](./docs/user/README.md) in `/docs/user`.
 
-### Example Template Fields
-
-```yaml
-spec:
-  moduleName: my-module
-  manager:
-    group: apps
-    version: v1
-    kind: Deployment
-    name: my-module-operator
-    namespace: my-module-system
-  data:
-    apiVersion: example/v1
-    kind: MyModule
-    metadata:
-      name: my-module
-  info:
-    repository: https://github.com/example/my-module
-    documentation: https://docs.example.com
-    icons:
-    - name: logo
-      link: https://example.com/icon.png
-```
-
----
-
-## Community Module Catalog
-
-All `ModuleTemplate` CRs are aggregated and published as:
-
-- [🔗 all-modules.yaml](https://kyma-project.github.io/community-modules/all-modules.yaml)
-- [🔗 all-modules.json](https://kyma-project.github.io/community-modules/all-modules.json)
-
-These endpoints are consumed by Kyma Dashboard to render a catalog of installable community modules.
-
-You can also use these files to install all modules at once:
-
-```bash
-kubectl apply -f https://kyma-project.github.io/community-modules/all-modules.yaml
-```
-
----
-
-## How to Contribute
-
-We welcome community module contributions! Follow these steps:
-
-1. Fork this repository
-2. Create a folder under `modules/your-module-name/`
-3. Add a valid `moduletemplate.yaml` (based on `v1beta2` spec)
-4. Open a pull request with a short description
-
----
+For more information on managed SAP BTP, Kyma runtime, see [Kyma Environment](https://help.sap.com/docs/btp/sap-business-technology-platform/kyma-environment?version=Cloud).
 
 ## Known Limitations
 
-- No SLA or upgrade guarantees for community modules
-- Incompatible or invalid templates may break during Dashboard rendering
-- Some modules may require additional configuration or permissions
-
+- Community modules aren't subject to the Service Level Agreement (SLA).
+- Incompatible or invalid ModuleTemplates may break during rendering in Kyma dashboard.
+- Some community modules may require additional configuration or permissions.
 
 ## Contributing
 
-For standard contribution rules see [CONTRIBUTING.md](CONTRIBUTING.md).
+We welcome community module contributions! To contribute, follow these steps:
+
+1. Fork the repository.
+2. Create a folder under `modules/your-module-name/`.
+3. Add a valid `moduletemplate.yaml` based on the `v1beta2` spec. For more information, see [`moduletemplate_types.go`](https://github.com/kyma-project/lifecycle-manager/blob/main/api/v1beta2/moduletemplate_types.go).
```

Comment:
@pbochynski, have I provided a valid link to the v1beta2 spec? Shall we keep it?